addons/core/functions/fnc_initKeybinds.sqf

```diff
@@ -114,56 +114,74 @@ TF_speak_volume_modifier_yelling_modifiers = [false, false, false];
-["TFAR","LRStereoLeft",    ["LR Stereo: Left","LR Stereo: Left"],{[1] call TFAR_fnc_processLRStereoKeys},    {true},[TF_lr_stereo_left_scancode, TF_lr_stereo_left_modifiers],false] call cba_fnc_addKeybind;
-["TFAR","LRStereoRight",    ["LR Stereo: Right","LR Stereo: Right"],{[2] call TFAR_fnc_processLRStereoKeys},{true},[TF_lr_stereo_right_scancode,TF_lr_stereo_right_modifiers],false] call cba_fnc_addKeybind;
-
-
-["TFAR","SWTransmit",["SW Transmit","SW Transmit"],{call TFAR_fnc_onSwTangentPressed},{call TFAR_fnc_onSwTangentReleased},[TF_tangent_sw_scancode, TF_tangent_sw_modifiers],false] call cba_fnc_addKeybind;
-["TFAR","SWTransmitAlt",["SW Transmit Alt","SW Transmit Alt"],{call TFAR_fnc_onSwTangentPressed},{call TFAR_fnc_onSwTangentReleased},[TF_tangent_sw_2_scancode, TF_tangent_sw_2_modifiers],false] call cba_fnc_addKeybind;
-["TFAR","SWTransmitAdditional",["SW Transmit Additional","SW Transmit Additional"],{call TFAR_fnc_onAdditionalSwTangentPressed},{call TFAR_fnc_onAdditionalSwTangentReleased},[TF_tangent_additional_sw_scancode, TF_tangent_additional_sw_modifiers],false] call cba_fnc_addKeybind;
-
-["TFAR","LRTransmit",["LR Transmit","LR Transmit"],{call TFAR_fnc_onLRTangentPressed},{call TFAR_fnc_onLRTangentReleased},[TF_tangent_lr_scancode, TF_tangent_lr_modifiers],false] call cba_fnc_addKeybind;
-["TFAR","LRTransmitAlt",["LR Transmit Alt","LR Transmit Alt"],{call TFAR_fnc_onLRTangentPressed},{call TFAR_fnc_onLRTangentReleased},[TF_tangent_lr_2_scancode, TF_tangent_lr_2_modifiers],false] call cba_fnc_addKeybind;
-["TFAR","LRTransmitAdditional",["LR Transmit Additional","LR Transmit Additional"],{call TFAR_fnc_onAdditionalLRTangentPressed},{call TFAR_fnc_onAdditionalLRTangentReleased},[TF_tangent_additional_lr_scancode, TF_tangent_additional_lr_modifiers],false] call cba_fnc_addKeybind;
-
-
-["TFAR","LowerHeadset",["Lower Headset","Lower Headset"],{},{true call TFAR_fnc_setHeadsetLowered;},[0, [false, false, false]],false] call cba_fnc_addKeybind;
-["TFAR","RaiseHeadset",["Raise Headset","Raise Headset"],{},{false call TFAR_fnc_setHeadsetLowered;},[0, [false, false, false]],false] call cba_fnc_addKeybind;
+_swCH1 = format ["%1 %2",localize "STR_TFAR_key_SW_Channe","1"];
+_swCH2 = format ["%1 %2",localize "STR_TFAR_key_SW_Channe","2"];
+_swCH3 = format ["%1 %2",localize "STR_TFAR_key_SW_Channe","3"];
+_swCH4 = format ["%1 %2",localize "STR_TFAR_key_SW_Channe","4"];
+_swCH5 = format ["%1 %2",localize "STR_TFAR_key_SW_Channe","5"];
+_swCH6 = format ["%1 %2",localize "STR_TFAR_key_SW_Channe","6"];
+_swCH7 = format ["%1 %2",localize "STR_TFAR_key_SW_Channe","7"];
+_swCH8 = format ["%1 %2",localize "STR_TFAR_key_SW_Channe","8"];
+["TFAR","SWChannel1",[_swCH1,_swCH1],{[0] call TFAR_fnc_processSWChannelKeys},{true},[TF_sw_channel_1_scancode, TF_sw_channel_1_modifiers],false] call cba_fnc_addKeybind;
```

Comment:
@dedmen 
I rewrite this with format, please check.

Reply:
Should probably make this a small local function.
You have a typo in the string name the `l` is missing.
```
_fnc_localizeSWChannel = {
format ["%1 %2",localize "STR_TFAR_key_SW_Channel", _this]
};
```
and then just `8 call _fnc_localizeSWChannel`

Or even
```
_fnc_localizeSWChannel = {
private _str = format ["%1 %2",localize "STR_TFAR_key_SW_Channel", _this];
[_str, _str]
};
```
so you only call it once per channel and get the two values from the array back

Reply:
Ok, I will rewrite this.